extension.ts

```diff
@@ -334,19 +327,27 @@ export async function activate(context: vscode.ExtensionContext) {
           case 'underline':
             options.cursorStyle = vscode.TextEditorCursorStyle.Underline;
             break;
-
           default:
             break;
         }
         editor.options = options;
       });
-      await vscode.commands.executeCommand('setContext', 'vim.active', Globals.active);
+      await vscode.commands.executeCommand('setContext', 'vim.active', false);
```

Comment:
Do we need to call this in two different spots (line 335, line 339)? You can probably consolidate it to outside the `if` to use `Configuration.disableExtension`

```
await vscode.commands.executeCommand('setContext', 'vim.active', Configuration.disableExtension)
```